pythonbytes.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
```

Comment:
I somewhat understand the thought process of having one separate file for one website so it would be easy to download, but I would rather go in the direction of code reuse and one unified interface, which can download from all kinds of websites. We should make this a proper package and reuse as much code as possible instead of duplicating it. I like to shoot on long-term maintainability and usability.